understanding/22/accessible-authentication.html

```diff
@@ -0,0 +1,120 @@
+ 
+        <p>Cognitive function tests are known to be problematic for many people with cognitive disabilities. Whether it is remembering random strings of characters, a pattern gesture to perform on a touch screen, or identifying which images include a particular object, this form of test will exclude some people. When a cognitive function test is used, that at least one other authentication method must available which uses does not use a cognitive function test.</p>
+            
+        <p class="note">Websites can employ username (or email) and password inputs as an authentication method if it meets <a href="#">Success Criterion 1.3.5 Input Purpose</a> and password managers / browser functionality is not blocked. When properly marked up, browser features or password managers can save the user’s information and refill the login.</p>
+
+        <p>If there is more than one step in the authentication process, such as with multi-factor authentication, all steps should comply with this success criterion. There should be a path through authentication that does not rely on cognitive function tests.</p>
+
+
+
+    </section>
+    <section id="benefits">
+      <h2>Benefits of Accessible Authentication</h2>
+      
+      <p>People with cognitive issues to do with memory, reading (e.g. dyslexia), numbers (e.g. dyscalculia), or perception-processing limitations will be able to authenticate irrespective of the level of their cognitive abilities.</p>
+      
+   </section>
+   
+   <section id="examples">
+      <h2>Examples of Accessible Authentication</h2>
+
+      <ul>
+         <li>A website uses a properly marked up email and password as the login authentication.</li>
+         <li>A website uses WebAuthn so the user can authenticate with their device instead of username/password. The user’s device could use any available modality. Common methods on laptops and phones are facial-scan, fingerprint, and pin number. The website is not enforcing any particular use, it is assumed a user will setup a method that suits them.</li>
+         <li>A website offers the ability to login with a 3rd party provider using the OAuth method.</li>
+         <li>A website that requires 2-factor authentication allows for multiple options for the 2nd factor, including a USB-based method where the user simply presses a button to enter a time-based token.</li>
+      </ul>
+      
+   </section>
+
+   
+   <section id="techniques">
+      <h2>Techniques for Accessible Authentication</h2>
+      
+      
+      <section id="sufficient">
+         <h3>Sufficient Techniques for Accessible Authentication</h3>
+         
+         
+         <ol>
+            <li>
+                <a href="#" class="general">Email link authentication</a>
+            </li>
+            <li>
+                @@ Providing a properly marked up email and password inputs.
+            </li>
+            <li>
+                @@ Providing WebAuth as an alternative to username/password.
```

Comment:
1. typo? "WebAuth"
2. these should be clearly phrased as a list of examples, rather than as currently phrased b/c it appears to be a definitive and finite list.

Reply:
sufficient techniques (as well as failure techniques) are all just non-exhaustive throughout WCAG understanding docs, i believe...they're just suggestions at that particular point in time, as there's usually many more ways to satisfy the ask of any particular SC

Reply:
Updated to WebAuthn. Sufficient techniques are always "one possible way" of meeting the guideline.